controller/lib/include/enumeration.h

```diff
@@ -284,6 +284,12 @@ enum notifications /// Notifications for the AVDECC library implementation, not
     TOTAL_NUM_OF_NOTIFICATIONS = 7
 };
 
+enum acmp_notifications
+{
+    BROADCAST_RESPONSE_RECEIVED = 0,
```

Comment:
Can we please insert NULL_ACMP_NOTIFICATION = 0 at the start of the list.
Also should be BROADCAST_ACMP_RESPONSE_RECEIVED because this is in the same namespace as other notifications.....